packages/admin/admin/src/mui/menu/CollapsibleItem.styles.ts

```diff
@@ -0,0 +1,60 @@
+import { Theme } from "@mui/material";
+import { createStyles } from "@mui/styles";
+
+import { MenuCollapsibleItemProps } from "./CollapsibleItem";
+
+export type MenuCollapsibleItemClassKey =
+    | "root"
+    | "childSelected"
+    | "listItem"
+    | "open"
+    | "collapsedMenuParentTitle"
```

Comment:
Is this being used anywhere? If not, please remove the key and it's styling.

Reply:
✅